### ISSUE TITLE
Issues with wiewing someones profile/comments - doesnt continue to load after reaching "bottom" even though theres more comments available

### ISSUE THREAD

**Issue by bngs** [QuantumBadger/RedReader#561]
I think it only loads the first page or a set amount of comments? Any possible fix or maybe a new option?

**Comment by QuantumBadger** [comment]
Yeah, at the moment it's only designed to fetch one page. It would be good to improve this though.

**Comment by bngs** [comment]
I've been skimming through the classes, but I cant find the class that handles user profiles. Mind pointing me in the right direction?

**Comment by QuantumBadger** [comment]
Sure :) The user profile dialog itself is here:

https://github.com/QuantumBadger/RedReader/blob/master/src/main/java/org/quantumbadger/redreader/fragments/UserProfileDialog.java

The lists of posts/comments are shown in:

https://github.com/QuantumBadger/RedReader/blob/master/src/main/java/org/quantumbadger/redreader/activities/CommentListingActivity.java

https://github.com/QuantumBadger/RedReader/blob/master/src/main/java/org/quantumbadger/redreader/activities/PostListingActivity.java

The actual logic for fetching and displaying them is in:

https://github.com/QuantumBadger/RedReader/blob/master/src/main/java/org/quantumbadger/redreader/fragments/CommentListingFragment.java

https://github.com/QuantumBadger/RedReader/blob/master/src/main/java/org/quantumbadger/redreader/fragments/PostListingFragment.java

We should already fetch an unlimited number of posts -- it's just the comments which are fixed.

Let me know if you have any other questions!

**Comment by nascentt** [comment]
This would really be nice to have.  I frequently have needed to go back through my comments to find old conversations and redreader just doesn't support it.

I've always used a secondary Reddit app to make up for the shortcoming but obviously won't be able to do so now.
